packages/server/src/services/eth-gas.ts

```diff
@@ -49,6 +49,7 @@ export class EthGasStream extends EventEmitter {
         let ws = EthGasStream.ws;
 
         if (!ws) {
+            console.log('[WS:GAS]: Opening connection to gasnow.org gas price oracle...');
```

Comment:
didnt see this land, are we not polling for gas prices anymore?

Reply:
Yeah added this over the weekend - I connected to a diff API which provides a websocket https://github.com/PeggyJV/il-loss-charts/pull/128